source/uwp/Renderer/lib/XamlHelpers.cpp

```diff
@@ -359,44 +359,48 @@ namespace AdaptiveCards::Rendering::Uwp::XamlHelpers
         return S_OK;
     }
 
-    void ApplyBackgroundToRoot(_In_ IPanel* rootPanel,
-                               _In_ IAdaptiveBackgroundImage* backgroundImage,
-                               _In_ IAdaptiveRenderContext* renderContext,
-                               _In_ IAdaptiveRenderArgs* renderArgs)
+    void CreateBackgroundImage(_In_ IAdaptiveRenderContext* renderContext, _In_ HSTRING url, _Outptr_ IImage** backgroundImage)
```

Comment:
nit: HSTRING const& url

Reply:
@licanhua
![image](https://user-images.githubusercontent.com/21356912/145633566-34190d54-cda0-458c-ae5f-3783f5ff950a.png)
there's no need for const& :)